src/main/java/frc/robot/Robot.java

```diff
@@ -5,30 +5,61 @@
 package frc.robot;
 
 import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.wpilibj.PowerDistribution;
 import edu.wpi.first.wpilibj.SPI;
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
 /**
  * The VM is configured to automatically run this class, and to call the functions corresponding to
  * each mode, as described in the TimedRobot documentation. If you change the name of this class or
  * the package after creating this project, you must also update the build.gradle file in the
  * project.
  */
 public class Robot extends TimedRobot {
-    public static boolean debug = false;
-    public static final AHRS navx = new AHRS(SPI.Port.kMXP);
+    public static final boolean debug = true;
+    private static final AHRS navx = new AHRS(SPI.Port.kMXP);
+    private static Rotation2d startAngle = new Rotation2d();
+    //    private final Compressor compressor = new Compressor(0);
     public PowerDistribution pdp = new PowerDistribution();
-    private RobotContainer m_robotContainer;
     private Command m_autonomousCommand;
+    private RobotContainer m_robotContainer;
+
+    /**
+     * Gets the current angle of the robot in respect to the start angle.
+     *
+     * @return the current angle of the robot in respect to the start angle.
+     */
+    public static Rotation2d getAngle() {
+        Rotation2d rotation = Rotation2d.fromDegrees(Math.IEEEremainder(Robot.navx.getRotation2d().getDegrees(), 360));
```

Comment:
You don't need IEEEremainder

Reply:
I keep it to be easier to read when printing.

Reply:
How the hack it's easier to read?

Reply:
when you want to print the rotation, it gives you values between -180 to 180 instead of being continuous to huge values.

Reply:
Yeah, because you use the `minus` function you don't need to normalize (try `System.out.println(Rotation2d.fromDegrees(9000).minus(Rotation2d.fromDegrees(30)).getDegrees());` for instance)

Reply:
right